docs/themes/godocs-2/layouts/_default/index.json

```diff
@@ -1,36 +1,5 @@
-{{- $len := sub (len (.Pages.GroupBy "Section")) 1 -}}
-[{{- range $item, $el := .Pages.GroupBy "Section" -}}
-{{- range $i, $e := .Pages -}}
-{{- if and (not .Params.ignoreSearch) (ne $e.Type "json") -}}
-  {
-    "section": "",
-    "url": "{{ $e.Permalink }}",
-    "title": "{{ with $e.Params.bannertext }}{{htmlEscape .}}{{else}}{{htmlEscape $e.Title}}{{end}}",
-    "description": "{{ htmlEscape .Description}}",
-    "searchKeyword": "{{ htmlEscape .Params.searchKeyword}}",
-    "content": {{$e.Plain | jsonify}}
-  },
-{{- end -}}
-
-{{- if .Section -}}
-{{- $section := (where site.Pages "Section" .Section) | intersect (where site.Pages ".Title" "!=" .Title) -}}
-{{- $sectionLen := len (where site.Pages "Section" .Section) -}}
-{{- $sectionLenTotal := sub $sectionLen 2 -}}
-
-{{- range $index, $page := $section -}}
-{{- if and (not .Params.ignoreSearch) (ne $page.Type "json") -}}
-  {
-    "section": "{{$page.Section | humanize}}",
-    "url": "{{ $page.Permalink }}",
-    "title": "{{ with $page.Params.bannertext }}{{htmlEscape .}}{{else}}{{htmlEscape $page.Title}}{{end}}",
-    "description": "{{ htmlEscape .Description}}",
-    "searchKeyword": "{{ htmlEscape .Params.searchKeyword}}",
-    "content": {{$page.Plain | jsonify}}
-  }{{- if eq $item $len -}}{{- if ne $index $sectionLenTotal -}},{{- end -}}{{- else -}},{{- end -}}
-{{- end -}}
-{{- end -}}
-
-{{- end -}}
-
-{{- end -}}
-{{- end -}}]
\ No newline at end of file
+{{ $index := slice }}
```

Comment:
This code tells Hugo to create an `index.json` file with the site's content. I didn't notice any `tags` so I removed that from line 3. However, if tags are added this can be modified by inserting `"tags" .Paramstags`. 

By the way, to view the JSON file you can visit `http://localhost:1313/index.json`.